src/components/JitsiMeeting.tsx

```diff
@@ -36,6 +36,7 @@ const JitsiMeeting = ({
   userInfo,
   spinner,
```

Comment:
you could use aliasing to uppercase spinner in the component's params instead of creating a new const on the spinner render function : {...,spinner: Spinner, ...} .Think it's cleaner this way.